addons/lamp/CfgEventHandlers.hpp

```diff
@@ -0,0 +1,44 @@
+class Extended_PreInit_EventHandlers
+{
+	class ADDON
+	{
+        // Like the normal preinit above, this one runs on all machines
+        init = call compile preprocessFileLineNumbers "\z\ae3\addons\network\XEH_preInit.sqf";
```

Comment:
Wrong reference. Should be "\z\ae3\addons\lamp\XEH_preInit.sqf" and not "\z\ae3\addons\network\XEH_preInit.sqf"

Reply:
fixed